### ISSUE TITLE
FontAwesome icon in button is not shrink

### ISSUE THREAD

**Issue by Dudeplayz** [FlowingCode/FontAwesomeIronIconset#43]
The default vaadin icons getting shrink  if they are slotted in a button:

```css
[part] ::slotted(vaadin-icon[icon^='vaadin:']), [part] ::slotted(iron-icon[icon^='vaadin:']) {
    padding: 0.25em;
    box-sizing: border-box !important;
}
```

Because the selector is checking for the `vaadin:` prefix, it doesn't apply to the FontAwesome icons.

**Comment by javier-godoy** [comment]
Thanks for reporting this issue. Fix has been released in version 4.1.0.

**Comment by Dudeplayz** [comment]
Thanks for the feedback :)
